Kickstarter-iOS/Views/Controllers/BackingViewController.swift

```diff
@@ -72,6 +74,14 @@ internal final class BackingViewController: UIViewController {
     _ = self.backerShippingDescriptionLabel |> UILabel.lens.font .~ .ksr_body()
     _ = self.backerAvatarImageView |> UIImageView.lens.accessibilityElementsHidden .~ true
 
+    _ = self.loadingIndicatorView
+      |> UIActivityIndicatorView.lens.animating .~ true
+      |> UIActivityIndicatorView.lens.activityIndicatorViewStyle .~ .white
+      |> UIActivityIndicatorView.lens.color .~ .ksr_navy_900
+
+    _ = self.loadingOverlayView
+      |> UIView.lens.backgroundColor .~ UIColor(white: 1.0, alpha: 0.99)
```

Comment:
we chatted irl about this but i think you can get rid of this since you did it in the storyboard